web/pandas/about/roadmap.md

```diff
@@ -15,10 +15,53 @@ fundamental changes to the project that are likely to take months or
 years of developer time. Smaller-scoped items will continue to be
 tracked on our [issue tracker](https://github.com/pandas-dev/pandas/issues).
 
-See [Roadmap evolution](#roadmap-evolution) for proposing
-changes to this document.
+The roadmap is defined as a set of major enhancement proposals named PDEPs.
+For more information about PDEPs, and how to submit one, please refer to
+[PEDP-1](/pdeps/accepted/0001-puropose-and-guidelines.html).
 
-## Extensibility
+## PDEPs
```

Comment:
I know this is bikeshedding, and certainly okay with it as-is, but maybe the abbreviation should be pdEP? My thinking here is:

 - With the "p" in pandas traditionally lowercase; as opposed to the P in Python (PEP) and N in NumPy (NEP)
 - Reflects `import pandas as pd`
 - I instinctively see "PDEP" and "P-DEP"

Reply:
I'd personally try to avoid adding extra complexity with the capitalization. I don't know what P-DEP is or what stands for. If it's something we want to avoid, my preference would be to use something totally different. Like PCP or PDCP (C for Change instead of E for ENHANCEMENT), or anything else. But if people prefer pdEP it, I'll update it.

Reply:
why not just PEP ; i don't think this conflicts with python enhancements and it's simpler and easier to pronounce

Reply:
PEP could be hard to find using search engines given the ubiquity of PEP as Python Enhancement Proposal.